build/sourcelink.props

```diff
@@ -0,0 +1,15 @@
+<Project>
+
+  <PropertyGroup>
+    <!-- enable on TeamCity -->
+    <SourceLinkCreate Condition="'$(SourceLinkCreate)' == '' and '$(BUILD_NUMBER)' != ''">true</SourceLinkCreate>
```

Comment:
This should make it work on your TeamCity server by default. It would be awesome if your could try it and send me a link to a nupkg file.